app/models/survey.rb

```diff
@@ -0,0 +1,138 @@
+class Survey
+  include ActionView::Helpers::SanitizeHelper
+  include ApplicationHelper
+
+  def initialize(faculty_survey)
+    @faculty_survey = faculty_survey
+    @survey_to_update = @faculty_survey.config_questions.order(:id)
+  end
+
+  def uniqname
+    @faculty_survey.uniqname
+  end
+
+  def unit_id
+    @faculty_survey.unit_id
+  end
+
+  def questions_to_display
+    if @faculty_survey.program_id.present?
+      survey = []
+      @survey_to_update.each do |item|
+        unless rich_text_value(item.question).include?("title") ||
+              rich_text_value(item.question).include?("not a course") ||
+              rich_text_value(item.question).include?("subject") ||
+              rich_text_value(item.question).include?("catalog") ||
+              rich_text_value(item.question).include?("section") ||
+              rich_text_value(item.question).include?("number of students using ride share") 
+          survey << item
+        end
+      end
+    else
+      survey = @faculty_survey.config_questions.order(:id)
+    end
+    return survey
+  end
+
+  def update_answers(params)
+    result = { 'success' => true, 'note' => '' }
+    course = false
+    # answers have rich_text format and params are something like this:
```

Comment:
delete commented code

Reply:
@rsmoke This is not the code, these are comments about the input format - to make it easier to understand why in the following code I had to use split, strip_tags, strip, what is 'item', etc.
```  
if p[0].split("_").first == "item"
    question_number = p[0].split("_").last
    question_id = p[0].split("_").second
    if question_number == '1' 
        if strip_tags(p[1]).strip == '' 
...
```

But if the code is understandable without comments I can delete them. Please let me know if you insist on deleting the comments.